test/api/thermostats/update.test.ts

```diff
@@ -33,4 +33,22 @@ test("POST /thermostats/update with api key", async (t) => {
   )
 
   t.is(20, device?.properties.default_climate_setting.heating_set_point_celsius)
+
+  // Test only manual_override_allowed
+  const { status: status2 } = await axios.post(
```

Comment:
FWIW, for the fake, we can prefer to have isolated tests vs. overloading single tests. They have zero cost compared to tests in projects that use an actual DB.